src/index.js

```diff
@@ -1,12 +1,28 @@
-import _ from 'lodash';
+import Task from './modules/Task.js';
+import { list } from './modules/elements.js';
 import './style.css';
+import listItem from './modules/list-item';
 
-function component() {
-  const element = document.createElement('div');
+const init = () => {
+  const task1 = new Task('wash the dishes');
+  const task2 = new Task('complete to Do list project');
```

Comment:
- [ ] Good job so far 👍 but you are to create an array of your tasks and not a class. Your tasks array should contain task objects which have a description, completed, and index keys. Kindly address this issue 💯 
- Requirements:
![Screenshot 2022-07-07 090348](https://user-images.githubusercontent.com/85113567/177724800-7a99e453-addd-463d-b6bd-0e1c6a8cbeb7.png)